QUARKUS-2994.md

```diff
@@ -0,0 +1,27 @@
+# QUARKUS-2994 JDK 21 Support
+
+This document is a high level test plan for certification of Red Hat build of Quarkus on JDK 21.
+
+## Scope of the testing
+
+### Impact on test suites and testing automation
+* Temurin JVM 21 and OpenJDK 21 need to be added as JDKs to Quarkus QE Jenkins
+* Temurin JVM 21 and OpenJDK 21 have to be added as JVMs to Polarion
+* Bare metal jobs in extended platform testing need to be extended to run with both Temurin 21 and OpenJDK 21.
+  * This affects JVM mode only.
+
+### Impact on resources
+* 1 large executor for bare metal testing for 4 hours on Temurin 21
+* 1 large executor for bare metal testing for 4 hours on OpenJDK 21
+
+## Future considerations
+* Since updating JDKs require additional effort on both ours and infra team side, we may have to come up with setting up the JDK ourselves
```

Comment:
These JDKs also need to be updated and both updates and installations are requested in Jira by the team, that needs it the most. We didn't do it previously, simply because we were lucky.

Reply:
So based on historic data, how much time do you think we'll save if we manage those JDKs ourselves instead of using the ones in `/qa/tools`? I don't have a strong opinion on this being here, I just don't think this has been a concern for us

Reply:
This change will not safe us much time, but will give more control and make updates more predictable. And will change workflow from "look in our code -> ask around -> find jira project -> find similar issue and copy it" to much simpler "look into our code -> fix our code"